ajax/libs/videojs-overlay/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "videojs-overlay",
+  "description": "A plugin to display simple overlays during video playback.",
+  "author": "Brightcove, Inc.",
+  "license": "Apache-2.0",
+  "version": "1.1.4",
+  "main": "es5/plugin.js",
+  "keywords": [
+    "videojs",
+    "videojs-plugin"
+  ],
+  "filename": "videojs-overlay.min.js",
+  "npmName": "videojs-overlay",
```

Comment:
Let's move this line under `repository` field. Because `npmName` and `npmFileMap` are CDNJS auto-update config to the library. It will be useful to us to put them together.
It will seem like as the followings:
```js
  "repository": {
    "type": "git",
    "url": "https://github.com/brightcove/videojs-overlay.git"
  },
  "npmName": "videojs-overlay",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/*"
      ]
    }
  ]
```